packages/next/src/next-devtools/dev-overlay/components/devtools-panel/devtools-panel-tab/segments-explorer-tab.tsx

```diff
@@ -1,12 +1,15 @@
-import { PageSegmentTree } from '../../overview/segment-explorer'
-
 function SegmentsExplorer({
   routerType,
   page,
 }: React.HTMLProps<HTMLDivElement> & {
   routerType: 'app' | 'pages'
   page: string
 }) {
+  if (!process.env.__NEXT_DEVTOOL_SEGMENT_EXPLORER) {
```

Comment:
If we want to dead code eliminate the below require it needs to have the if condition wrap the require